more_itertools/tests/test_more.py

```diff
@@ -177,6 +177,147 @@ def test_slicing_reset(self):
             next(p)
             eq_(p[index], seq[1:][index])
 
+    def test_passthrough(self):
+        """Iterating a peekable without using ``peek()`` or ``prepend()``
+        should just give the underlying iterable's elements (a trivial test but
+        useful to set a baseline in case something goes wrong)"""
+        expected = [1, 2, 3, 4, 5]
+        actual = list(peekable(expected))
+        eq_(actual, expected)
+
+    # prepend() behavior tests
+
+    def test_prepend(self):
+        """Test calling ``prepend()`` at various points interspersed with calls
+        to ``next()``"""
+        it = peekable(range(5))
+        it.prepend(10)
```

Comment:
Could these go in a loop? I'm usually OK with violating DRY in tests, but I think this would be easier to understand in a loop. That, or do this test with a smaller iterable?